apps/etterlatte-saksbehandling-ui/client/src/components/generelloppgave/GenerellOppgave.tsx

```diff
@@ -0,0 +1,107 @@
+import { Alert, Button, Heading, Textarea, TextField } from '@navikt/ds-react'
+import React, { useEffect } from 'react'
+
+import styled from 'styled-components'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useSidetittel } from '~shared/hooks/useSidetittel'
+import { opprettGenerellOppgave } from '~shared/api/oppgaver'
+import { GenerellOppgaveDto, OppgaveKilde, Oppgavetype } from '~shared/types/oppgave'
+import { mapAllApiResult } from '~shared/api/apiUtils'
+import { ApiErrorAlert } from '~ErrorBoundary'
+import { useFeatureEnabledMedDefault } from '~shared/hooks/useFeatureToggle'
+
+export default function GenerellOppgave() {
+  useSidetittel('Opprett generell oppgave')
+  const [generellOppgaveResult, opprettNyGenerellOppgave] = useApiCall(opprettGenerellOppgave)
+  const skalViseSide = useFeatureEnabledMedDefault('opprette-generell-oppgave', false)
+
+  const methods = useForm<GenerellOppgaveDto>({
+    defaultValues: {
+      merknad: '',
+      sakIds: '',
+      type: Oppgavetype.GENERELL_OPPGAVE,
+      kilde: OppgaveKilde.BEHANDLING,
+    },
+  })
+
+  const { reset } = methods
+
+  useEffect(() => {
+    if (generellOppgaveResult.status === 'success') {
+      reset()
+    }
+  }, [generellOppgaveResult, reset])
+
+  const opprett = (data: GenerellOppgaveDto) => {
+    opprettNyGenerellOppgave({
+      sakIds: data.sakIds,
+      merknad: data.merknad,
+      type: data.type,
+      kilde: data.kilde,
+    })
+  }
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = methods
+
+  return (
+    <>
+      {skalViseSide && (
+        <FormWrapper>
+          <FormProvider {...methods}>
+            <Heading size="large">Opprett generell oppgave</Heading>
+
+            <TextField
+              {...register('sakIds', {
+                required: { value: true, message: 'Du må oppgi minst en saks-ID' },
+              })}
+              label="Saker"
+              description="Legg til saks-ID-er i en kommaseparert liste"
+              placeholder="sakid1, sakid2, ...."
+              error={errors.sakIds?.message}
+            />
```

Comment:
Burde dette kanskje heller være en liste med tekst felter for hver sak ID man skal ha med? Tror det blir mer brukervenlig når de slipper å måtte forholde seg til å komma sepparere hver sak ID

Reply:
Men igjen så vet ikke jeg hvor ofte saksbehandlerne kommer til å opprette generelle oppgaver. Så det er kan funke som førsteutkast

Reply:
Ja helt enig, vi har pratet om det og vert litt frem og tilbake. Vi landet på å teste det ut as-is. Utfordringen er hvis f.eks 20+ saker skal av overstyrt-beregning og dette må legges inn en og en. Er noen grupper i overstyrt-beregning som er store.

Reply:
Burde i så fall gjøre split i frontend. Blir mye ryddigere kode å skrive: 

`sakIds: data.sakIds.split(',').map((id) => Number(id))`

... i stedet for de greiene her: 

``` 

                    val sakIds: List<Long> =
                        oppgaveBulkDto.sakIds
                            .split(
                                ",",
                            ).map { it.trim() }
                            .filter { it.isNotEmpty() }
                            .map { it.toLong() }
```